Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -712,8 +712,8 @@ When a segmented image is given as input, the generated mesh surface sometimes s
 to the voxels surface, causing an aliasing effect.
 A solution to generate a smooth and accurate output surface was described by Stalling et al in
 \cgalCite{stalling1998weighted}. It consists in generating a second input image, made
-of integer coefficients called "weights". This image can be generated using
-`CGAL::Mesh_3::generate_weights()` as shown in
+of integer coefficients called *weights*, and use those weights to define smoothing domain boundaries.
```

Comment:
"*smoothing* domain boundaries": That does not sound right, does it?

Reply:
indeed. I meant *smoother*